### PR TITLE
chore(site): remove dead code

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
Close https://github.com/coder/coder/issues/9599

File: site/package.json [hunk 1]
```diff
@@ -161,14 +159,12 @@
...
     "prettier": "3.0.0",
     "protobufjs": "7.2.4",
-    "resize-observer": "1.0.4",
     "rxjs": "7.8.1",
     "ssh2": "1.14.0",
     "storybook": "7.4.0",
     "storybook-addon-react-router-v6": "2.0.0",
     "storybook-react-context": "0.6.0",
     "ts-node": "10.9.1",
-    "ts-proto": "1.157.0",
```
Comment (code-asher) [inline site/package.json:171]: Looks like `ts-proto` is used in `make gen` (via `pnpm run gen:provisioner`)  so we might need to add this one back.

**Review by code-asher — APPROVED** [review]
The best kind of PR!  Not sure about `ts-proto` but the rest looks good to me.

File: site/package.json (hunk 1 above)
Comment (BrunoQuaresma) [reply site/package.json:171]: Yes!

**Comment by code-asher** [comment]
I think maybe the lockfile is supposed to be generated by `pnpm` instead of regular `npm`?  Not exactly sure but CI is saying:

```
ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up to date with package.json
```

**Review by mtojek — COMMENTED** [review]
> The best kind of PR! Not sure about ts-proto but the rest looks good to me.

FTR `ts-proto` is used by `gen:provisioner` to generate the provisioner stub. The imports from stub are used to build "echo" supported templates. [Link](https://github.com/coder/coder/blob/main/site/e2e/helpers.ts#L7-L17)

**Pull request merged**